lib/jsdom/living/events/EventTarget-impl.js

```diff
@@ -234,6 +241,48 @@ function invokeEventListeners(listeners, target, eventImpl) {
 
 const wrappedListener = Symbol("inline event listener wrapper");
 
+/**
+ * Normalize the event listeners options argument in order to get always a valid options object
+ * @param   {Object} options         - user defined options
+ * @param   {Array} defaultBoolKeys  - boolean properties that should belong to the options object
+ * @param   {Array} keysBlackList    - options not allowed (optional)
+ * @returns {Object} object containing at least the  "defaultBoolKeys"
+ */
+function normalizeEventHandlerOptions(options, defaultBoolKeys, keysBlackList) {
+  // normalize the options argument in order to get always an object
+  if (options === null || options === undefined) {
+    options = {};
+  } else if (typeof options !== "object") {
+    options = { capture: options };
+  }
+
+  let defaultOptions;
+  let additionalOptions;
+  keysBlackList = keysBlackList || [];
+
+  try {
```

Comment:
That test is specifically testing that any custom option values are *not* gotten. An error is thrown if they ever are gotten.